pymc3/tests/test_plots.py

```diff
@@ -0,0 +1,54 @@
+import matplotlib.pyplot as plt
```

Comment:
I assume from the creation of this new file that `plot_posterior_predictive_glm` wasn't tested anywhere before?

Reply:
that's right